src/app/services/company/company.service.ts

```diff
@@ -35,4 +35,16 @@ export class CompanyService extends BaseService {
         const url = `${environment.backend.url}${Company.URL}/activation/${activationToken}`;
         return this.request(METHOD.POST, url, httpOptions);
     }
+
```

Comment:
Acho que era fixe agruparmos estes dois métodos talvez para o serviço `auth.service.ts`

Reply:
a escolha de por no `company.service.ts` foi porque está lá o save company com a password a ser criado e acesso ao endpoint do activation também. No BE pusemos no resource da Company também. Achas que mesmo assim devemos mudar para `auth.service.ts`?

Reply:
Eu também fiquei na duvida 😛
A minha ideia era separar lógica de criação de resources das de autenticação.
No nosso caso o resource que representa o `AuthUser` é a `Company` mas também podia ser um `User`.
Se tivermos a lógica de CRUD dos resources isolada nos seus "service" e autenticação num "service" dedicado, para, eventualmente, trocarmos o resource de `AuthUser` era mais fácil. E tínhamos separação de responsabilidades também 😉

Reply:
Migrado para `AuthUserService`